backend/prisma/migrations/20240628180651_data_model_v1/migration.sql

```diff
@@ -0,0 +1,23 @@
+-- CreateTable
+CREATE TABLE "event" (
```

Comment:
Just wondering if the file name is intented, `migrations/20240628180651_data_model_v1/migration.sql` sounds kind of weird

Reply:
Yes, its the name that prisma gives to the migration. I will try to change it to see what happens